bundles/CoreBundle/PimcoreCoreBundle.php

```diff
@@ -57,6 +58,18 @@ public function getContainerExtension()
         }
     }
 
+    public function boot()
+    {
+        parent::boot();
+
+        /**
+         * BC: Remove with Pimcore 6.0.0
+         */
+        if ($this->container->hasParameter('pimcore.mime.extensions')) {
```

Comment:
This is only for the case that someone tries to access `Mime::$extensionMapping` directly in project/bundle code, right? 
Do you think that's commonly used? Imho it shouldn't, since this was always intended to be an internal variable.

Reply:
Imho we should remove this BC layer and make a note in upgrade notes. 
Can you please update your PR? Thank you very much!

Reply:
I don't think anyone directly access the variable, but I wasn't sure to be honest. I will just remove the bc layer and the variable and create a Upgrade note.

Reply:
Great, thanks!

Reply:
done